transport/grpc/doc.go

```diff
@@ -51,6 +75,28 @@
 //     },
 //   })
 //
+// To make requests using TLS to an application supporting gRPC over TLS, pass
+// credentials.TransportCredentials as a DialerCredentials DialOption. There
+// are various ways to create credentials.TransportCredentials. See
+// https://godoc.org/google.golang.org/grpc/credentials#TransportCredentials.
+//
+//   myTLSConfig := &tls.Config{
+//     // any arbitrary valid tls.Config
+//   }
+//   myTransportCredentials := credentials.NewTLS(myTLSConfig)
+//   myDialOptions := []DialOption{DialerCredentials(myTransportCredentials)}
+//   myChooser := peer.NewSingle(
+//     hostport.Identify("127.0.0.1:4443"),
+//     t.NewDialer(myDialOptions...),
```

Comment:
Same: I would put the DialerCredentials option here directly instead of putting it in a variable.

Reply:
done